docs/DeveloperGuide.md

```diff
@@ -633,13 +633,49 @@ Similar to [<ins>UC09](#use-case-uc09-add-phone-number-to-contact) except duplic
 * 2a.The tag list is empty.
   Use case ends.
 
-* 3a. The provided tag name does not exist.
-    * 3a1. The system displays an error message.
+* 3a. The contact already has the specified tag assigned to them.
+    * 3a1. System displays an error message.
+  
+      Use case ends.
 
-      Use case resumes at step 1.
+* 3b. User specified to force the tagging of a contact.
```

Comment:
Nit but maybe "specifies" so not past tense?

Reply:
```
**MSS**

1. User <ins>lists all contacts [(UC01)](#use-case-uc01-list-all-contacts)</ins>.
2. User <ins>lists all tags [(UC04)](#use-case-uc04-list-all-tags)</ins>.
3. User requests to tag a contact with a specified tag.
4. System displays the updated contact with the new tag assigned to them, in the list of contacts.

   Use case ends.

**Extensions**

* 1a. The contact list is empty.
  Use case ends.

* 2a.The tag list is empty.
  Use case ends.

* 3a. The contact already has the specified tag assigned to them.
    * 3a1. System displays an error message.
  
      Use case ends.

* 3b. User specified to force the tagging of a contact.
    * 3b1. System displays the newly added tag, in the tag list. 
```

for this i think maybe specified might be more appropriate, because it is referring to the request the user carried out in step 3, not sure though